packages/wrangler/src/experimental-assets.ts

```diff
@@ -39,10 +39,11 @@ type UploadResponse = {
 
 // constants same as Pages for now
 const BULK_UPLOAD_CONCURRENCY = 3;
-const MAX_ASSET_COUNT = 20_000;
-const MAX_ASSET_SIZE = 25 * 1024 * 1024;
 const MAX_UPLOAD_ATTEMPTS = 5;
 const MAX_UPLOAD_GATEWAY_ERRORS = 5;
+// NB also used in miniflare in plugins/kv/assets.ts, so please update there too.
+const MAX_ASSET_COUNT = 20_000;
+const MAX_ASSET_SIZE = 25 * 1024 * 1024;
```

Comment:
Strawman proposal: export these from `workers-shared` and add that as a devDep here.

Reply:
went a bit overboard with that here... 
https://github.com/cloudflare/workers-sdk/commit/9f3bf38bd769a29b080435a26aa5b638eb8d48aa 
(doesn't build and I think I've gotten confused, but could this work?)